recipe/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "streamlit-card" %}
+{% set version = "0.0.4" %}
```

Comment:
The latest version is 0.0.42 https://github.com/gamcoh/st-card/compare/v0.0.4...v0.0.42

Reply:
The latest version (and the one before it) are not on PyPi. I switched to pulling down the archives from the GitHub releases, but they don't test successfully (even the v0.0.4 tag). `v0.0.4` satisfies the need, so I would opt that we stick with this.